src/test/unit/platform.h

```diff
@@ -43,7 +43,7 @@
 
 
 #define PID_PROFILE_COUNT 4
-#define CONTROL_RATE_PROFILE_COUNT  4
+#define CONTROL_RATE_PROFILE_COUNT 6
```

Comment:
No need to change this, or reduce number to 2 (if 2 profiles are enough for unittests).
```suggestion
#define CONTROL_RATE_PROFILE_COUNT 4
```